nav2_map_server/src/map_saver_cli.cpp

```diff
@@ -41,31 +41,31 @@ int main(int argc, char ** argv)
     } else if (*it == "-f") {
       if (++it == arguments.end()) {
         RCLCPP_WARN(logger, "Argument ignored: -f should be followed by a value.");
-        continue;
+        break;
       }
       params_from_args.emplace_back("output_file_no_ext", *it);
     } else if (*it == "--occ") {
       if (++it == arguments.end()) {
         RCLCPP_WARN(logger, "Argument ignored: --occ should be followed by a value.");
-        continue;
```

Comment:
This shouldn’t fail silently. This should probably exit the process with code `-1`

Reply:
You wouldn’t want a false attempt at an argument fall back to defaults. That would give the user unexpected behavior in their file saving process which might be the only map of a SLAM session.

Reply:
@SteveMacenski okay. then i can exit the codes. But it seems that it doesn't have consistency on other value. In case of other input param, it just skips to process. 
ex) ./map_saver --testargignored 
==> ignore and run

Do you intend to exit all other case in cli?

Reply:
That’s a good point, yes, the default case of a non-valid argument should also fail. 

The warning messages for all of them should change to say exiting, not ignoring, because of the issue. The message should be elevated to `RCLCPP_ERROR`.

Reply:
@SteveMacenski okay. then i will apply it.